build.sbt

```diff
@@ -95,7 +95,12 @@ lazy val interfaces = crossProject(JVMPlatform, NativePlatform)
       IO.write(props, "scalafmt properties", out)
       List(out)
     },
-  ).jvmSettings(crossVersion := CrossVersion.disabled, autoScalaLibrary := false)
+  ).jvmSettings(
+    javacOptions ++= Seq("-source", "8", "-target", "8"),
+    Compile / doc / javacOptions := Seq(),
```

Comment:
are you sure you still need this for doc? especially if you used the one above with `Compile /`?

Reply:
yeah, it still didn't work without it